src/app/frontend/replicasetlist/replicasetlist.scss

```diff
@@ -0,0 +1,31 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+$replicaset-deploy-btn: #eb3a2e;
```

Comment:
You know what? How about we delete this color from here and use default accent palette? This means removing `let kubernetesAccentPaletteName = 'kubernetesAccentPallete';` from `index_config.js`. 

This will make us stock angular material app, which is good. I remember us modifying accent palette to make md-tab different from default, but I think this is not needed unless seriously justified. 

What do you think? @floreks @cheld @zreigz